src/Symfony/Component/ErrorHandler/ErrorRenderer/CliErrorRenderer.php

```diff
@@ -108,4 +183,23 @@ private function findExceptionFilePathAndName(string $filepath): array
 
         return [$dirPath, $fileName];
     }
+
+    private function makePathRelative(string $absolutePath): string
+    {
+        $vendorPrefix = str_replace([
+            'symfony'.DIRECTORY_SEPARATOR.'error-handler'.DIRECTORY_SEPARATOR.'ErrorRenderer'.DIRECTORY_SEPARATOR.'CliErrorRenderer.php',
+            'src'.DIRECTORY_SEPARATOR.'Symfony'.DIRECTORY_SEPARATOR.'Component'.DIRECTORY_SEPARATOR.'ErrorHandler'.DIRECTORY_SEPARATOR.'ErrorRenderer'.DIRECTORY_SEPARATOR.'CliErrorRenderer.php',
+        ], '', __FILE__);
+        $projectPrefix = \dirname($vendorPrefix).DIRECTORY_SEPARATOR;
+
+        return str_replace($projectPrefix, '', $absolutePath);
+    }
+
+    private function renderAsLink(string $content, string $filePath, int $lineNumber): string
+    {
+        $terminalSupportsLinks = 'JetBrains-JediTerm' !== getenv('TERMINAL_EMULATOR') && (!getenv('KONSOLE_VERSION') || (int) getenv('KONSOLE_VERSION') > 201100);
+        $href = sprintf('file://%s#L%d', $filePath, $lineNumber);
+
+        return $terminalSupportsLinks ? "\033]8;;{$href}\033\\{$content}\033]8;;\033\\" : $content;
```

Comment:
would there be no way to extract this bit? This is repeated from https://github.com/symfony/symfony/blob/5a2ee5a9d1817b565acc4e35e1ac601cc8142b4e/src/Symfony/Component/VarDumper/Dumper/CliDumper.php#L445-L446 too

Reply:
Repeating that isn't ideal - add a `CliDumper::handlesHrefGracefully()` public static function?